index.html

```diff
@@ -59,6 +59,11 @@
                 <img class="nav__image" src="./images/flag-es.png" alt="Spanish" />
               </a>
             </li>
+            <li class="nav__list-item">
+              <a class="nav__link" href="./ru/">
+                <img class="nav__image" src="./images/flag-ru.png" alt="Russian" />
+              </a>
+            </li>
```

Comment:
@TatianaFokina could you put it before spanich to keep alphabetical order?

Reply:
@TatianaFokina please add this navigation item to `/es/index.html` and `/jp/index.html`